config/rbac/role.yaml

```diff
@@ -184,6 +184,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumes
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
nit - appreciate they are all pretty much the same but do we need `list` and `watch` here?

Reply:
So the ctrl runtime client will attempt to cache anything you GET once,
and for that, you need list and watch:
https://sdk.operatorframework.io/docs/faqs/#after-deploying-my-operator-i-see-errors-like-failed-to-watch-external-type

Reply:
+1